executor/builder.go

```diff
@@ -4192,7 +4192,7 @@ func (b *executorBuilder) buildWindow(v *plannercore.PhysicalWindow) Executor {
 	partialResults := make([]aggfuncs.PartialResult, 0, len(v.WindowFuncDescs))
 	resultColIdx := v.Schema().Len() - len(v.WindowFuncDescs)
 	for _, desc := range v.WindowFuncDescs {
-		aggDesc, err := aggregation.NewAggFuncDesc(b.ctx, desc.Name, desc.Args, false)
```

Comment:
Why we call it twice?

Reply:
when doing the plan, the agg funcs have called TypeInfer, but here executing pahse, buildWindow do this again. In fact, we should not change the plan after final plan.